src/util/safe_print.h

```diff
@@ -42,6 +42,7 @@
 #include <time.h>
```

Comment:
Happy for me to scope creep and change this to `<ctime>` (as recommended to me by @tkoeppe)?